sample-deployment/sample-deployment-web-app/package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "dummy-website",
+  "version": "1.0.0",
+  "main": "index.js",
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "description": "",
+  "dependencies": {
+    "body-parser": "1.18.2",
+    "canvg": "2.0.0",
+    "cors": "2.8.1",
+    "cytoscape": "2.7.14",
+    "d3-v4-grid": "2.0.1",
+    "dotenv": "8.2.0",
+    "errorhandler": "1.5.0",
+    "express": "4.16.0",
+    "fs": "0.0.1-security",
+    "googleapis": "1.1.5",
+    "http": "0.0.0",
+    "jspdf": "1.5.3",
+    "method-override": "2.3.10",
+    "moment": "2.24.0",
+    "morgan": "1.9.1",
+    "multiparty": "4.2.2",
+    "node-xlsx": "0.7.2",
+    "nodemon": "1.19.1",
+    "parallelshell": "3.0.1",
+    "pg": "8.0.3",
+    "pg-hstore": "2.3.3",
+    "pug": "3.0.2",
+    "querystring": "0.2.0",
+    "save-svg-as-png": "1.4.14",
+    "sequelize": "5.21.6",
+    "serve-static": "1.13.1",
+    "should": "11.2.0",
+    "stylus": "0.54.5",
+    "supertest": "3.0.0",
+    "url": "0.11.0",
+    "webpack": "4.0.0",
+    "xml2js": "0.4.17",
+    "xmlbuilder": "8.2.2"
+  },
+  "devDependencies": {
+    "browser-env": "3.3.0",
+    "chai": "4.1.2",
+    "coveralls": "2.13.1",
+    "eslint": "3.19.0",
+    "isomorphic-fetch": "2.2.1",
+    "istanbul": "0.4.5",
+    "jquery": "3.6.0",
+    "jquery-extend": "2.0.3",
+    "jsdom": "13.2.0",
+    "markdown-pdf": "8.0.0",
+    "mocha": "2.5.3",
+    "mocha-lcov-reporter": "1.3.0",
+    "nock": "10.0.6",
+    "sinon": "5.1.1",
+    "w3c-xmlserializer": "1.0.1",
+    "webpack-cli": "2.0.14",
+    "xmlserializer": "0.6.1"
+  },
+  "babel": {
+    "presets": [
+      "es2015"
+    ]
+  },
+  "scripts": {
+    "webpack-start": "webpack --watch",
+    "test": "mocha --reporter list --compilers js:babel-core/register",
+    "start-dev": "parallelshell --wait \"nodemon web-client/app.js\" \"nodemon server/app.js\" \"webpack --watch --mode=development\"",
+    "start-prod": "parallelshell --wait \"nodemon web-client/app.js\" \"nodemon server/app.js\" \"webpack --watch --mode=production\"",
+    "coverage": "istanbul cover _mocha",
+    "lint": "eslint \"**/*.js\"",
+    "coveralls": "npm run coverage -- --report lcovonly && cat ./coverage/lcov.info | coveralls"
```

Comment:
We’ll need a script here that also runs the static web server, per the comment above